### ISSUE TITLE
bun install fails on slow registries

### ISSUE THREAD

**Issue by carneiror** [oven-sh/bun#4799]
### What version of Bun is running?

1.0.0+822a00c4d508b54f650933a73ca5f4a3af9a7983

### What platform is your computer?

Darwin 22.6.0 arm64 arm

### What steps can reproduce the bug?

```
➜  bun-test time bun install axe-core
bun add v1.0.0 (822a00c4)
  🔍 axe-core [6/6]
error: Timeout downloading package manifest axe-core
error: axe-core@ failed to resolve
bun install axe-core  0.05s user 0.14s system 0% cpu 6:00.05 total
```

### What is the expected behavior?

```
➜  bun-test time npm install axe-core
npm install axe-core  4.31s user 0.53s system 3% cpu 2:20.91 total
```

### Additional information

This registry and the package in question are extremely slow (18mb and take for than 1min to download). Bun do 6 retries, 1 minute each and fails every time.

I tried set higher values for [fetch-retry-maxtimeout](https://docs.npmjs.com/cli/v10/using-npm/config#fetch-retry-maxtimeout), [fetch-retry-mintimeout](https://docs.npmjs.com/cli/v10/using-npm/config#fetch-retry-mintimeout) and [fetch-timeout](https://docs.npmjs.com/cli/v10/using-npm/config#fetch-timeout) without any success.

This is an issue with my current private registry but should be also an issue for people with really bad connections.

**Comment by tomconder** [comment]
+1 my use case is similar: timeout fetching axe-core from a slow private registry.

**Comment by Jarred-Sumner** [comment]
Fixed by #6217
